main.go

```diff
@@ -235,7 +236,7 @@ func main() {
 	flag.IntVar(&rowsPerRequest, "rows-per-request", 1, "clustering rows per single request")
 	flag.BoolVar(&provideUpperBound, "provide-upper-bound", false, "whether read requests should provide an upper bound")
 	flag.BoolVar(&inRestriction, "in-restriction", false, "use IN restriction in read requests")
-	flag.BoolVar(&selectOrderByDesc, "select-order-by-desc", false, "if true will add 'order by ck desc' to the select queries")
+	flag.StringVar(&selectOrderBy, "select-order-by", "none", "controls order part 'order by ck desc' of the read query, you can set it to: none,asc,desc or to the list of them, i.e. 'none,asc', in such case it will be randomized over these values")
```

Comment:
perhaps will be clearer to change: "controls order part 'order by ck desc'  ->
"controls order part 'order by ck <asc/desc>'

Reply:
fixed